backend/src/main/java/de/laura/project/api/model/TriviaApiData.java

```diff
@@ -0,0 +1,20 @@
+package de.laura.project.api.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class TriviaApiData {
+
+    private String category;
+    private String type;
+    private String difficulty;
+    private String question;
+    private String correct_answer;
+    private String[] incorrect_answers;
```

Comment:
try using `List<String>` instead of `String[]`
https://stackoverflow.com/questions/49012470/consuming-a-nested-json-array-using-spring-boot-and-resttemplate